back/app/models/activity.rb

```diff
@@ -15,10 +15,10 @@
 #
 # Indexes
 #
-#  index_activities_on_acted_at               (acted_at)
-#  index_activities_on_action                 (action)
-#  index_activities_on_item_type_and_item_id  (item_type,item_id)
-#  index_activities_on_user_id                (user_id)
+#  index_activities_on_acted_at  (acted_at)
+#  index_activities_on_action    (action)
+#  index_activities_on_item      (item_type,item_id)
```

Comment:
Why is this change here?

Reply:
Basically, both are the same. `schema.rb` was updated 7 months ago in 342e7c8a7c7b3a1199c030cddcea6c2037560606
Maybe nobody just committed the change in the annotations before.

Reply:
Also, it was changed to the wrong value here 57d514483b6798bffd41e1548786b18425b7732a
I'm pretty sure by mistake.